javascriptv3/example_code/cloudwatch-events/README.md

```diff
@@ -1,73 +1,66 @@
-# Amazon CloudWatch Events JavaScript SDK v3 code examples
-## Purpose
+# CloudWatch Events code examples for the SDK for JavaScript in Node.js
+
+## Overview
+
 The code examples in this directory demonstrate how to work with Amazon CloudWatch Events
```

Comment:
(nit) one line per sentence for consistency with other markdown

Reply:
Noted. Not convinced though.